05-js-add-objects/index.js

```diff
@@ -0,0 +1,88 @@
+/*  ----- NECESSARY TASK SOLUTION ---- */
+function add(...array) {
+  let result = {};
+  array.forEach(el => {
```

Comment:
This is normal solution but not readable. Can you try to write more readable variant without result variable and without using indexes?

Reply:
ok, I will try to see into it